pkg/api/graph/test/restarting-pod.yaml

```diff
@@ -74,5 +86,108 @@ items:
           reason: 'Image: openshift/ruby-hello-world is not ready on the node'
     phase: Pending
     startTime: 2015-07-13T19:36:05Z
+- apiVersion: v1
+  kind: Pod
+  metadata:
+    annotations:
+      kubernetes.io/created-by: |
+        {"kind":"SerializedReference","apiVersion":"v1","reference":{"kind":"ReplicationController","namespace":"default","name":"gitlab-ce-1","uid":"f74e7dce-c6ad-11e5-9bbb-080027c5bfa9","apiVersion":"v1","resourceVersion":"120461"}}
+      openshift.io/deployment-config.latest-version: "1"
+      openshift.io/deployment-config.name: gitlab-ce
+      openshift.io/deployment.name: gitlab-ce-1
+      openshift.io/generated-by: OpenShiftNewApp
+      openshift.io/scc: restricted
+    creationTimestamp: 2016-01-29T17:30:21Z
+    generateName: gitlab-ce-1-
+    labels:
+      app: gitlab-ce
+      deployment: gitlab-ce-1
+      deploymentconfig: gitlab-ce
+    name: gitlab-ce-1-lc411
+    namespace: example
```

Comment:
It has to be tested.  Why are you running this as an example?  Use oc
export if you need to clear the namespace.